src/main/kotlin/com/londogard/summarize/embeddings/WordEmbeddings.kt

```diff
@@ -198,22 +197,7 @@ fun <T> PriorityQueue<T>.addR(element: T): PriorityQueue<T> {
 }
 
 /** Aggregate (sum) the given list of vectors
- * @param vecs The input vector(s).
+ * @param vectors The input vector(s).
  * @return The sum vector (aggregated from the input vectors).
  */
-fun sumVector(vectors: List<Array<Float>>): Array<Float> = vectors.reduce { agg, itr -> agg + itr }
-
-
-object RunWord2Vec {
-    @JvmStatic
-    fun main(args: Array<String>) {
-        val model = WordEmbeddings(dimensions =  50)
-
-        // distance: Find N closest words
-        model.distance(listOf("frace"), N = 10)?.let(model::pprint)
-        model.distance(listOf("france", "usa"))?.let(model::pprint)
-        model.distance(listOf("france", "usa", "usa"))?.let(model::pprint)
-        model.analogy("king", "queen", "man", N = 10)?.let(model::pprint)
-        model.rank("apple", setOf("orange", "soda", "lettuce", "pear")).let(model::pprint)
-    }
-}
\ No newline at end of file
+fun sumVector(vectors: List<Array<Float>>): Array<Float> = vectors.reduce { agg, itr -> agg + itr }
```

Comment:
`private`

Reply:
Would be nice with better names in reduce?